docs/troubleshooting.md

```diff
@@ -94,11 +95,11 @@ This should trigger the same error as the one found in `docker logs` from the pr
 /usr/bin/nsjail --config /config/nsjail.cfg -Mo -v
 ```
 
-This should output more errors, that should provide you with more details about what went wrong.
+This should output more errors, and these errors should provide more details about what went wrong.
 
 ## Troubleshooting with Kubernetes
 
-If everything works in docker, the problem might be higher up (in Kubernetes). The first step to debug this would be to check if the challenge works in a local cluster. Follow the instructions [here](local-testing.md#running-the-challenge-in-kubernetes) for getting the task running in KIND.
+If everything works in Docker, the problem might be higher up (in Kubernetes). The first step to debug this would be to check if the challenge works in a local cluster. Follow the instructions [here](local-testing.md#running-the-challenge-in-kubernetes) for getting the task running in KIND.
```

Comment:
for KIND, maybe link to https://github.com/kubernetes-sigs/kind

Reply:
Done